gulpfile.js

```diff
@@ -1,45 +1,65 @@
-const {src, dest, watch} = require('gulp');
+//Constantes globales a utilizar
+const {src, dest, watch, series} = require('gulp');
```

Comment:
add space after `src` and before `series` something like this: `const { src, dest, watch, series }`